spec/unit/rack/user_spec.rb

```diff
@@ -189,5 +189,19 @@ def current_user
         expect(user_data).to be_empty
       end
     end
+
+    context "when Rack user field getters require parameters" do
+      class UserRequiringParameters
+        def name(hello)
+          "my name is #{hello}"
+        end
+      end
+
+      let(:user_data) { described_class.new(UserRequiringParameters.new).as_json }
+
+      it "is empty" do
```

Comment:
Would rephrase to make the intention more explicit. For example, "does not call the methods with arguments".